e2e/e2e.js

```diff
@@ -469,3 +698,141 @@ describe('delete', () => {
     expect(res.statusText).toBe('OK')
   })
 })
+
+// Test organization integrations (similar to workspace credentials), commented out
```

Comment:
is this one of the bugs you filed? (maybe have a reference to internal bug number)

Reply:
yup IOC-4291